### PR TITLE
Document why version ranges won't work

### PR DESCRIPTION
**Pull request by DavidBoike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +86,15 @@ As we don't care about patching older releases, the Platform Sample uses a simpl
   * Patch releases generally should be avoided. If one is necessary, the release branch would need to be created from the point of the labeled minor, and the patch released from there.
 * Once a release is built on master, promote to Deploy, which sends it to Octopus and MyGet.
 * Deploy to NuGet by promoting in Octopus.
+
+### Why not use version ranges / wildcard dependency?
+
+Projects using packages.config have control over whether to resolve the **highest matching** or **lowest matching** dependency. However this feature has not yet been added for `PackageReference` in SDK style project files in Visual Studio 2017 and higher.
+
+If, for example, Particular.PlatformSample were to reference Particular.PlatformSample.ServiceControl (>= 4.0.0 && < 5.0.0), then on a project using the platform sample NuGet would resolve the **lowest matching** version within that range. The effect would be that updated Particular.PlatformSample.ServiceControl would not automatically be deployed in projects unless the end user explicitly updated that package to latest.
```
Comment (bording) [inline README.md]:
```suggestion
If, for example, Particular.PlatformSample were to reference Particular.PlatformSample.ServiceControl (>= 4.0.0 && < 5.0.0), then, on a project using the platform sample, NuGet would resolve the **lowest matching** version within that range. The effect would be that the updated Particular.PlatformSample.ServiceControl would not automatically be deployed in projects unless the end user explicitly updated that package to latest.
```

File: README.md [hunk 2]
```diff
@@ -86,3 +86,15 @@ As we don't care about patching older releases, the Platform Sample uses a simpl
...
+
+For that reason, a new minor release of Particular.PlatformSample must (unfortunately) be released every time one of the underlying packages is updated
```
Comment (bording) [inline README.md]:
```suggestion
For that reason, a new minor release of Particular.PlatformSample must (unfortunately) be released every time one of the underlying packages is updated.
```

**Comment by bording** [comment]
A couple minor tweaks, otherwise LGTM

**Comment by DavidBoike** [comment]
Tweaks applied.

**Pull request merged**